### ISSUE TITLE
Error when cookies contain double quotes

### ISSUE THREAD

**Issue by na--** [loadimpact/k6#711]
While testing something else, I stumbled on a small bug with double quotes in cookie values... The following code:
```js
import http from 'k6/http';

export default function () {
	let resp = http.get("https://httpbingo.org/cookies", {
		"cookies": {
			"test": "\""
		},
	})
	console.log(resp.body);
}
```
produces this... warning?:
```
2018/07/16 10:18:33 net/http: invalid byte '"' in Cookie.Value; dropping invalid bytes
```
It seems that for some reason, the Go standard library [logs this directly](https://github.com/golang/go/blob/22e17d0ac7db5321a0f6e073bd0afb949f44dd70/src/net/http/cookie.go#L385), which is super annoying... Skimming some [golang issues](https://github.com/golang/go/issues/18627), it's not exactly clear how we should escape those values...

**Comment by ofauchon** [comment]
Hello, 

Same problem for me !

The application I test uses double quoted cookies values with separators + uppercase headers

Cookie: foo-boa="DATA-NET-CMP:1234564:123456789789798AEZEAZEA:zzzX"
SPECIFIC-HEADER=jfdislfjsdi

I ended up patching net/* go sources  to make it permissive :-( 
- Removed cookie/headers sanitization . 

Although it' dirty hacks, I didn't find any elegant way to do it.


Olivier

**Comment by na--** [comment]
Yeah, when I posted this issue I skimmed a bunch of golang github issues and discussions and I don't think I saw a nice way to work around this...

**Comment by nishant-shah-social** [comment]
Hello All,
Is there any workaround to this?
I am currently blocked because of this issue. We need to set a cookie in our performance test script whose value contains double quotes and its getting escaped :(

**Comment by rbayliss** [comment]
Try URL encoding the string as a workaround. Eg:
```
http.cookieJar().set('https://www.google.com', 'my_cookie', encodeURIComponent("\"I have quotes!\""));
```
